config/locales/doc_auth/en.yml

```diff
@@ -89,6 +94,16 @@ en:
       text4: make sure you always have access
       welcome: 'What you''ll need to do:'
     tips:
+      document_capture_header_text: 'For best results:'
```

Comment:
I notice that in #3990, I'd put these under the `instructions` heading, since that's where the existing `take_pic` texts had been located. I think objectively, they make more sense as "tips" as you've organized them here. Maybe it was an intentional change on your part. I'd be fine with them being under tips, but in reconciling with #3990, we should update references I'd added.

Reply:
I believe I have reconciled them under `tips` -- if you want to take a quick look to make sure I didn't miss anything...

Reply:
Looks good to me 👍